Example/PrebidDemo/PrebidDemoSwift/BannerController.swift

```diff
@@ -333,6 +352,28 @@ class BannerController:
             self?.mpBanner.loadAd()
         }
     }
+    
+    func loadAdMobRenderingBanner() {
+        gadBanner.delegate = self
+        appBannerView.addSubview(gadBanner)
+        gadBanner.backgroundColor = .red
+        gadBanner.rootViewController = self
+        
+        
+        let size = CGSize(width: 320, height: 50)
+        
+        mediationDelegate = AdMobMediationBannerUtils(gadRequest: gadRequest, bannerView: gadBanner)
+        prebidAdMobMediaitonAdUnit = MediationBannerAdUnit(configID: "50699c03-0910-477c-b4a4-911dbe2b9d42", size: size, mediationDelegate: mediationDelegate)
+        
+        prebidAdMobMediaitonAdUnit.fetchDemand { [weak self] result in
+            let extras = GADCustomEventExtras()
+            let prebidExtras = self?.mediationDelegate.getEventExtras()
+            extras.setExtras(prebidExtras, forLabel: "PrebidAdMobCustomEvent")
```

Comment:
Need to introduce the constant for "PrebidAdMobCustomEvent". And let's rename it to "PrebidAdMobCustomEventExtras" since it serves as a key for prebid's extras data.